omnibus/config/software/erlang-crypto2.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2012-2017 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "erlang-crypto2"
+default_version "master"
+
+source git: "https://github.com/jaym/erlang-crypto2.git"
+
+# TODO get Jay to convert this to Apache 2
+license "Reserved"
+license_file "LICENSE"
+
+dependency "erlang"
+dependency "rebar"
```

Comment:
@tyler-ball Chef Server uses rebar3 everywhere.  We moved to rebar3 during the beta so we vendored rebar3 into each app at the time of conversion.  We have a long-standing TODO to get to a single version now that it is stable. (You'll see src/oc_erchef/rebar3, src/bookshelf/rebar3, etc).